src/app/shared/services/applications/application.service.ts

```diff
@@ -39,12 +35,14 @@ export class ApplicationService {
         params = params.set('SearchString', parameters.searchString);
       }
 
+      if ('size' in parameters && 'from' in parameters) {
```

Comment:
Can we check this same as above? (`parameters.size && parameters.from`)

Reply:
I did it at first but faced with an issue. A parameter From can be equal "0", and this condition returns false but has to return true because this pagination's parameter exists. So, I decided to use this check.


![01](https://github.com/ita-social-projects/OoS-Frontend/assets/67392473/646384a2-1477-4a8f-b4ac-65313009c1a6)


If you tell me better solution, I'll be appreciate.

Reply:
In this case, we can use the initial solution